src/ec/packages/ec-component-card/demo/data--tile-taxonomy.js

```diff
@@ -0,0 +1,64 @@
+import adapter from '../adapter';
+
+const specsTileTaxonomy = {
```

Comment:
can't you import the specs file instead of duplicating it?

Reply:
I forgot that one, that's how i did it before the specs were available, sure!! :)